jpf-symbc/src/main/gov/nasa/jpf/symbc/numeric/solvers/ProblemChoco.java

```diff
@@ -324,8 +335,12 @@ public Boolean solve() {
         pb.getSolver().setTimeLimit(ProblemChoco.timeBound);
 
         Boolean result = pb.solve();
-//        if (result == null)
- //       	System.out.println("Choco PC"+pb.pretty());
+
+		if (result == null) {
+			throw new RuntimeException("# Error: Choco returned null (possibly due to timeout).\n" +
+					"Time limit: " + ProblemChoco.timeBound + "\n" +
+					"Problem state:\n" + pb.pretty());
+		}
```

Comment:
It is okay to raise an exception here, but as we discussed we should catch that when we have multiple solvers, we should not crash, but rather count on other solvers, i.e., z3 to give us an answer.